### DISCUSSION TITLE
[Feature] <Possibility to login by username>

### DISCUSSION THREAD

**Discussion by aimer63** [immich-app/immich#7845] [Feature Request]
### The feature

First, great appreciation for this wonderful project!

My feature request is the option to login by username, keeping also the option to use the mail or let the Admin decide.
- Mandatory giving away PI is not a best practice so email should be optional
- When you create the Admin account you already spend the mail there and cannot reuse it for a normal user.
- Easier management for small group, families...

I know there are workarounds like joe@joe...  

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #3189
